docs/globals.rst

```diff
@@ -168,16 +167,24 @@ Environment variables of ``MyFunction`` will be set to ``{ TABLE_NAME: "resource
             TABLE_NAME: resource-table
             NEW_VAR: hello
 
+In the above example the environment variables of ``MyFunction`` will be set to:
+
+.. code:: json
+
+  {
+    STAGE: Production, 
```

Comment:
Could you wrap these in double quotes for JSON?

Reply:
Sure

Reply:
Done